samples/sampler/stories/default/IncrementSlider.js

```diff
@@ -17,6 +17,7 @@ storiesOf('Agate', module)
 		() => (
 			<IncrementSlider
 				active={boolean('active', Config)}
+				buttonSize={select('buttonSize', ['small', 'large'], Config, 'large')}
```

Comment:
We don't need to define the `large` default value because the story book already know it from Config.
`decrementIcon={select('decrementIcon', decrementIcons, Config)}`

Reply:
Could you omit `horizontal` in `orientation={select('orientation', ['horizontal', 'vertical'], Config, 'horizontal')}` below with the same reason above?